src/taker/api.rs

```diff
@@ -865,14 +846,18 @@ impl Taker {
 
         let previous_maker = self.ongoing_swap_state.peer_infos.iter().rev().nth(1);
 
-        log::info!("Connecting to {}", this_maker.address);
+        log::info!(
+            "Connecting to {} | Send Sigs Init Next Hop",
```

Comment:
I kept this log for us devs, and they are the same name as the function they are being called into for easy reference for us. This could even be debug logs. The user doesn't need to read this.

Our logs are very messy and verbose right now. It could be a followup PR to separate the logs into INFO and DEBUG. Only show INFO logs for user runs and DBUG logs in our IT.

For now, I will keep these as it is. Their only purpose it identify if any of these loops ever go infinite.

Reply:
Yes. Whatever the user gets to see by default in cli log files, should be info logs.
Whatever extra logs we need for dev understandings, they should be marked debug logs.

That way we can display a sanitized log to user while having all sorts of weird logging for ourselves.

It will require some thoughts on choosing all the logs between info and debug.